.gitignore

```diff
@@ -8,3 +8,5 @@ scripts/*.h
 scripts/expected.go
 scripts/output.go
 magic/magic.go
+crit/images/*.pb.go
+crit/images/*.proto
```

Comment:
It would be good to add the `*.proto` files to avoid downloading them every time.